yamls/us/main.yaml

```diff
@@ -58,7 +58,8 @@
       - [0x6DBC0, asm]
       - [0x6DF40, asm] # Probably has at least 1 file split
       - [0x70160, asm]
-      - [0x70200, asm, fault]
+      - [0x70200, c, fault]
+      - [0x720B0, asm, padding720B0] # Better way?
```

Comment:
Not sure if there is a better way here. Basically `Fault_HungUp` ends on a `0x10` boundary, but there are still 4 extra nops right afterwards.